src/core/distributions/phylogenetics/substitution/AbstractPhyloCTMCSiteHomogeneous.h

```diff
@@ -680,6 +680,7 @@ void RevBayesCore::AbstractPhyloCTMCSiteHomogeneous<charType>::compress( void )
         std::map<std::string,size_t> patterns;
         for (size_t site = 0; site < num_sites; ++site)
         {
+            bool all_missing = true;
```

Comment:
here is also some new code that removes site columns that only contain missing entries. Per se, this should not be a problem. However, if the +I model is used, then we get illegal memory access because we multiply the probability with the base frequency of the observed invariant site. For an all gap column, we don't know this index, that we access the base frequency vector at a random position. See https://github.com/revbayes/revbayes/issues/336